### PR TITLE
Update configure-wd-app-guard.md (User recommendations)

### PR DESCRIPTION
**Pull request by ojrb**
I create this PR for the already solved issue #4408 adding the user recommendations after previous merge! please review again and merge

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ These settings, located at **Computer Configuration\Administrative Templates\Net
...
+
+|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: 1) Include a full domain name for example **www.contoso.com** in the configuration, to trust only in this literal value 2) You may optionally use "." as a wildcard character to automatically trust subdomains on only one level on the domain hierarchy, that is to the left of the dot. Configuring **".constoso.com"** will automatically trust **"local.contoso.com"**, **"shop.contoso.com"** and all other values on the left. 3)  You may optionally use too ".." as a wildcard character to automatically trust subdomains on all the levels of the domain hierarchy, that are to the left of the dot. Configuring **"..constoso.com"** will automatically trust **"us.shop.contoso.com"**, **"trainning.contoso.com"** and all other values on the left. |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md]:
```suggestion
|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: 1) Include a full domain name in the configuration, for example **www.contoso.com**, to trust only in this literal value 2) You may optionally use "." as a wildcard character to automatically trust subdomains on only one level on the domain hierarchy that is to the left of the dot. Configuring **".constoso.com"** will automatically trust **"local.contoso.com"**, **"shop.contoso.com"**, and all other values on the left. 3)  You may optionally use ".." as a wildcard character to automatically trust subdomains on all the levels of the domain hierarchy that are to the left of the dots. Configuring **"..constoso.com"** will automatically trust **"us.shop.contoso.com"**, **"trainning.contoso.com"**, and all other values on the left. |
```

**Comment by mypil** [comment]
@ojrb  - please merge the commit on the copy/edit review made by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by ojrb** [comment]
Ready ! Suggestions committed ! please review and merge !

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4408. 

Thank you.

File: windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md [hunk 2]
```diff
@@ -33,7 +33,9 @@ These settings, located at **Computer Configuration\Administrative Templates\Net
...
 |-----------|------------------|-----------|
 |Private network ranges for apps|At least Windows Server 2012, Windows 8, or Windows RT|A comma-separated list of IP address ranges that are in your corporate network. Included endpoints or endpoints that are included within a specified IP address range, are rendered using Microsoft Edge and won't be accessible from the Application Guard environment.|
-|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: 1) If you want to specify a complete domain, include a full domain name (for example "**contoso.com**") in the configuration. 2) You may optionally use "." as a previous wildcard character to automatically trust all subdomains (when there is more than one subdomain). Configuring "**.constoso.com**" will automatically trust "**subdomain1.contoso.com**", "**subdomain2.contoso.com**", etc. 3) To trust a subdomain, precede your domain with two dots, for example "**..contoso.com**". |
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md]: Adding a blank line here seems to break the table in rich text diff view. Please check and see if removing this blank line will resolve this.
```suggestion
```

File: windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md [hunk 3]
```diff
@@ -33,7 +33,9 @@ These settings, located at **Computer Configuration\Administrative Templates\Net
...
+|Enterprise resource domains hosted in the cloud|At least Windows Server 2012, Windows 8, or Windows RT|A pipe-separated (\|) list of your domain cloud resources. Included endpoints are rendered using Microsoft Edge and won't be accessible from the Application Guard environment. Notes: 1) Include a full domain name in the configuration, for example **www.contoso.com**, to trust only in this literal value. 2) You may optionally use "." as a wildcard character to automatically trust subdomains on only one level of the domain hierarchy that is to the left of the dot. Configuring **".constoso.com"** will automatically trust **"local.contoso.com"**, **"shop.contoso.com"**, and all other values on the left. 3) You may optionally use ".." as a wildcard character to automatically trust subdomains on all the levels of the domain hierarchy that are to the left of the dots. Configuring **"..constoso.com"** will automatically trust **"us.shop.contoso.com"**, **"trainning.contoso.com"**, and all other values on the left. |
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard.md]: Adding a blank line here seems to break the table in rich text diff view. Please check and see if removing this blank line will resolve this.
```suggestion
```

**Comment by mypil** [comment]
@ojrb  - Kindly apply the suggested changes made by @illfated 

Thank you.

**Comment by illfated** [comment]
... and as usual, the rich text view shows the effective result of the changes made to the text:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4722/files?short_path=a45bc7d#diff-a45bc7d66c8c8f8c82a34a30055c25fb

**Pull request merged**